### ISSUE TITLE
Application Freezes After Period of Time

### ISSUE THREAD

**Issue by utkonos** [x64dbg/x64dbg#2473]
- Debugger: Oct 2 2020 (GleeBug)
- OS: Windows 10 Pro 1903 18362.329

The debugger application locks up after a period of time. This does not appear to be dependent on the executable used. It happens consistently after a period of time using the debugger. The environment is VirtualBox 6.1.14 on macOS 10.15.7.

This behavior does not exist using VMWare Fusion. Is it reproducible on a different set of hardware running the same version of macOS. The operating system does not lock up. Keyboard works and ctrl-alt-delete works. Mouse is able to move, but nothing is clickable and focus does not follow the mouse. Using the keyboard to navigate to Task Manager and then ending debugger task, the mouse is still not usable. However, running the debugger again fresh gets everything working again and after that point the mouse is usable as normal.

**Comment by mrexodia** [comment]
I believe I reproduced this when I left x64dbg running for an extremely long time while tracing. Will try to reproduce again!

**Comment by utkonos** [comment]
Let me know if there is anything I can do to provide more information. Is there anything specific you would like me to do or information I should gather when this occurs? It happens very consistently for me.

**Comment by mrexodia** [comment]
Yeah perhaps use task manager to create a dump of the x64dbg process while
in a frozen state.
